packages/plugin-dts/src/tsc.ts

```diff
@@ -58,10 +62,10 @@ export function emitDts(
         logger.error(message);
       }
 
-      throw new Error('TypeScript compilation failed');
+      throw new Error('DTS generation failed');
```

Comment:
User `.d.ts.` instead of `DTS` in user side?

Reply:
How about using `declarations files` instead of `DTS`?

Reply:
we can add specific suffix file names after we support `autoExtensions` in DTS

Reply:
"DTS" does not seem to be a formal term. We can change it in the future based on how the log looks in the end.